db/python/layers/participant.py

```diff
@@ -620,6 +620,10 @@ async def get_external_participant_id_to_internal_sequencing_group_id_map(
             project=project, sequencing_type=sequencing_type
         )
 
+    async def participant_table_export(self, project: int):
```

Comment:
One more trivial comment: Other functions here (and similarly for _sample.py_ and over in _../table/*.py_) have names of the form `actionverb_object`, whereas this is `object_verb` or maybe `adjectivalphrase_noun`. Could this be `export_participant_table` or so instead, or is there something being conveyed by this difference?

Reply:
Ah thanks for picking that up, I found the naming for this feature a bit tricky generally as I'm not 100% sure how it'll be used, so I changed the names a bunch of times and must have ended up on something out of step with existing naming accidentally.